install.sh

```diff
@@ -1,11 +1,14 @@
 sudo apt update
 sudo apt install astyle cmake gcc ninja-build libssl-dev python3-pytest python3-pytest-xdist unzip xsltproc doxygen graphviz python3-yaml python3-pip git
-pip3 install cryptography
-pip3 install argon2-cffi
+sudo apt install python3.11-venv
```

Comment:
I guess this can be sudo apt install python3-venv , we should avoid fixing it to 3.11